ksql-engine/src/main/java/io/confluent/ksql/services/KafkaTopicClientImpl.java

```diff
@@ -244,31 +273,15 @@ public void deleteInternalTopics(final String applicationId) {
         deleteTopics(internalTopics);
       }
     } catch (final Exception e) {
-      log.error("Exception while trying to clean up internal topics for application id: {}.",
+      LOG.error("Exception while trying to clean up internal topics for application id: {}.",
           applicationId, e
       );
     }
   }
 
   private static boolean isTopicDeleteEnabled(final AdminClient adminClient) {
     try {
-      final DescribeClusterResult describeClusterResult = adminClient.describeCluster();
-      final Collection<Node> nodes = describeClusterResult.nodes().get();
-      if (nodes.isEmpty()) {
-        log.warn("No available broker found to fetch config info.");
-        throw new KsqlException("Could not fetch broker information. KSQL cannot initialize");
-      }
-
-      final ConfigResource resource = new ConfigResource(
-          ConfigResource.Type.BROKER,
-          String.valueOf(nodes.iterator().next().id())
-      );
-
-      final Map<ConfigResource, Config> config = ExecutorUtil.executeWithRetries(
-          () -> adminClient.describeConfigs(Collections.singleton(resource)).all().get(),
-          ExecutorUtil.RetryBehaviour.ON_RETRYABLE);
-
-      return config.get(resource)
+      return getConfig(adminClient)
           .entries()
           .stream()
           .filter(configEntry -> configEntry.name().equalsIgnoreCase("delete.topic.enable"))
```

Comment:
This is safe to switch to a straight get:

```suggestion
          .get(KafkaConfig.DeleteTopicEnableProp())
```